book/website/communication/personal.md

```diff
@@ -0,0 +1,42 @@
+(cm-personal)=
```

Comment:
Did you check for any references to the labels that have been updated to make sure links aren't broken?

Reply:
I would use `grep` or `ripgrep`, and `sed` to update any links.

like,

`grep '#cm-personal-websites' **/*.md`
and
`sed -i 's/#cm-personal-websites/#cm-personal/g' **/*.md`

Reply:
I think there are no links other than the ones on the page itself!